lesson-4/script.js

```diff
@@ -0,0 +1,74 @@
+let Watermelon = 30;
+console.log(`Watermelon: ${Watermelon} UAH`);
+let Carrot = 8;
+console.log(`Carrot: ${Carrot} UAH`);
+let Potato = 12;
+console.log(`Potato: ${Potato} UAH`);
+let userChoice = prompt("What would you like to buy?")
+const userChooseWatermelon = userMoney / Watermelon;
+// console.log(Math.floor(userChooseWatermelon));
+const userChooseWatermelonChange = userMoney % Watermelon;
+// console.log(userChooseWatermelonChange);
+const userChooseCarrot = userMoney / Carrot;
+// console.log(Math.floor(userChooseCarrot));
+const userChooseCarrotChange = userMoney % Carrot;
+// console.log(userChooseCarrotChange);
+const userChoosePotato = userMoney / Potato;
+// console.log(Math.floor(userChoosePotato));
+const userChoosePotatoChange = userMoney % Potato;
+// console.log(userChoosePotatoChange);
+if (userChoice == "Watermelon" && userMoney >= Watermelon) {
+    console.log(`За ${userMoney} грн Ви можете придбати ${Math.floor(userChooseWatermelon)} одиниць даного продукту. У Вас залишиться ${userChooseWatermelonChange} грн`);
+} else if (userChoice == "Watermelon" && userMoney < 8) {
+    console.log(`На жаль, у Вас недостатньо коштів для купівлі хоча б 1 шт. будь-якого продукту.`)
+} else if (userChoice == "Watermelon" && userMoney < Watermelon) {
+    console.log(`У вас недостатньо коштів, для купівлі хоча б 1 шт. ${userChoice}`)
+}
+if (userChoice == "Carrot" && userMoney >= Carrot) {
+    console.log(`За ${userMoney} грн Ви можете придбати ${Math.floor(userChooseCarrot)} одиниць даного продукту. У Вас залишиться ${userChooseCarrotChange} грн`);
+} else if (userChoice == "Carrot" && userMoney < 8) {
+    console.log(`На жаль, у Вас недостатньо коштів для купівлі хоча б 1 шт. будь-якого продукту.`)
+}
+ else if (userChoice == "Carrot" && userMoney < Carrot) {
+    console.log(`У вас недостатньо коштів, для купівлі хоча б 1 шт. ${userChoice}`)
+}
+if (userChoice == "Potato" && userMoney >= Potato) {
+    console.log(`За ${userMoney} грн Ви можете придбати ${Math.floor(userChoosePotato)} одиниць даного продукту. У Вас залишиться ${userChoosePotatoChange} грн`);
+} else if (userChoice == "Potato" && userMoney < 8) {
+    console.log(`На жаль, у Вас недостатньо коштів для купівлі хоча б 1 шт. будь-якого продукту.`)
+}
+ else if (userChoice == "Potato" && userMoney < Potato) {
+    console.log(`У вас недостатньо коштів, для купівлі хоча б 1 шт. ${userChoice}`)
+}
+let triangleSide1 = +prompt("Enter length of the first side of triangle.")
+let triangleSide2 = +prompt("Enter length of the second side of triangle.")
+let triangleSide3 = +prompt("Enter length of the third side of triangle.")
+if ((triangleSide2 + triangleSide3) > triangleSide1 && (triangleSide3 + triangleSide1) > triangleSide2 && (triangleSide2 + triangleSide1) > triangleSide3) {
+    console.log("Triangle exist!")
+} else {
+    console.log("Triangle doesn't exist!")
+}
+/* 18 HW poin
+
+2 && 0 && 3 || true && false // false;
+false || " " || 3 && true // true;
```

Comment:
```suggestion
false || " " || 3 && true // " ";
```

Reply:
here I've missed, that empty string - is not an empty string but, a space